r_markdown/ef_summary.Rmd

```diff
@@ -0,0 +1,82 @@
+---
+title: "Emission Factor Summary"
+output:
+  html_document:
+    toc: true
+    toc_float: true
+    theme: united
+---
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  library(tidyverse)
+  library(forcats)
+```
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  source("../r_scripts/R_functions.R")
+```
+
+## Load and tidy data
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  load("../r_files/emissions.Rda")  # load emissions data
+  load("../r_files/samples.Rda")  # load sample meta data
+  load("../r_files/fuel_burnt.Rda")  # fuel use meta data
+```
+
+* Output emissions column names 
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  colnames(emissions)
+```
+
+* Filter out bad tests
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  emissions <- dplyr::filter(emissions, qc != "bad")
+```
+
+* Merge with fuel burnt data
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  emissions <- dplyr::left_join(emissions, dplyr::select(fuel_burnt, id,
+                                                 mass_fuel, lhv,
+                                                 carbon_fraction), by = "id") %>%
+           dplyr::left_join(dplyr::select(samples, id, stove,
+                                          fuel, fuelcat, stovecat,
+                                          stove_fuel), by = "id")
+```
+
+## Calculate emissions factors
+ 
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  emissions <- dplyr::mutate(emissions, mass_ef = 
```

Comment:
use units for your new variables names like we've done for others (e.g. ppm, ug...), i.e g_kg...